docs/components/attention/react.md

```diff
@@ -98,6 +113,32 @@ function Example() {
 }
 ```
 
+### Accessibility
+
+The attention element handles accessibility automatically by wrapping its slotted content with a `div` that has a default `role`attribute (`role="tooltip"` for tooltip and `role="img"` otherwise), and a default localized `aria-label`.
+
+It is possible to reset the `role` and `aria-label` attributes, and the default `aria-label` can be overridden:
```

Comment:
Some small comments here:
- "Attention component" (not "element")
- "`role` attribute" (missing space in "`role`attribute")
- maybe it would make sense to add a note on `aria-details` here? You can also add a "read more [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details)" to not use too much energy on explaining what it does ;)

Reply:
Hmm we don't set the `aria-details` automatically in the attention-component for vue and react as we do for elements, so not sure what I should write regarding `aria-details` here 🙈

Reply:
My bad, I was a bit too quick with my comment and it came out wrong 😅 My point was, because we are not doing this automatically, I thought it might be good to instruct users on the use of `aria-details` on the target element, so that they remember to create a relationship between the Attention and its target if that makes sense in their context. But just to save time, maybe it would suffice to quickly mention it and send users to MDN docs for more info. Does that make sense?

Reply:
Ah ok! That makes sense! I’ll add some info about aria-details. :)